site/content/docs/03-run-time.md

```diff
@@ -475,13 +475,48 @@ TODO
 * fade, fly, slide, scale, draw
 * crossfade...
 
-### `svelte/animation`
+### `svelte/animate`
 
-TODO
+The `svelte/animate` module exports one function for use with svelte [animations](docs#Animations).
+
+#### `flip`
+
+```sv
+animate:flip={params}
+```
+
+The `flip` function animates elements from their start position to their end position across `x` and `y` axes.
```

Comment:
I use the link mentioned in the tutorial, https://aerotwist.com/blog/flip-your-animations/ and also use or replicate their very well done bullet list in the "The general approach" as honestly I was thinking 'flip' was involving rotations. Something along these lines:


• **F** (first) - the starting state 
• **L** (last) - the ending state
• **I** (invert) - at the start of the transition the element has indeed moved in the DOM but we show it in the same position it was beforew moving, applying an inverted transformation
• **P** (play) - finally we play the animation interpolating the state from start to end

Reply:
I don't really think this belongs in an API Ref (unless I misunderstand the goal). API Ref's are just what a thing does and how to use it not an indepth explanation of implementation details.

We also don't want to be duplicating the tutorial content.

Reply:
Well the goal is to document what "flip" means and since "flip" in English means rotation, somewhere we should say that it's an acronym, by linking an external source or explaining it, "flip" alone pretty much confused me (I still haven't read the whole tutorial).

Reply:
If you only read the title, I can understand that but it is immediately followed by a description of the function (animating the x and y values). If we need to link to external articles or explain implementation in order for users to understand a piece of the API then I would consider that a problem.

Reply:
I've changed my mind. I'm also improving the general description.